app/helpers/format_links_helper.rb

```diff
@@ -0,0 +1,138 @@
+require "cgi"
+
+module FormatLinksHelper
```

Comment:
If we're going to split this (the alternative being to just increase the limit as we've never worried about the rubocop length limits much) then I thing something like `BrowseTagsHelper` would be better given these helper methods are about dealing with formatting tag keys and values for the data browser rather than more general link formatting.

Reply:
Sounds good, this is renamed now.

I'm fairly new to Ruby and rubocop, so I don't yet have an opinion about the rubocop rules, I was just trying to resolve the cascade of offenses. (First I got ModuleLength, then when I split the module it led to AssignmentInCondition, although I realize I made this unclear by committing the fixes in reverse order.)

If you're not concerned about those offenses, I'm happy to leave them as they were.  In that case, we could just look at commit 5ba64ef for this feature.